templates/emails/order.thankyou.fr.hbs

```diff
@@ -36,6 +36,13 @@ Subject: Merci pour votre contribution de {{currency order.totalAmount currency=
   <a href="https://opencollective.com/{{collective.slug}}">https://opencollective.com/{{collective.slug}}</a>.
 </p>
 
+{{#if customMessage}}
+<p>Un message de {{collective.name}}:</p>
```

Comment:
@Betree : Not sure if this translation is correct? 🤔

Reply:
It is :+1: